### ISSUE TITLE
Fix headers in DJA light and dark mode

### ISSUE THREAD

**Issue by rachidatecs** [cisagov/manage.get.gov#3417]
### Current Behavior

<img width="1088" alt="Image" src="https://github.com/user-attachments/assets/fc2daf41-5195-4209-8d28-9008135b1500" />

We set default header colors for our app, and as a side effect it overrode header colors too.

### Expected Behavior

DJA header color should be white

### Steps to Reproduce

1. Log into admin

### Additional Context

Make sure the admin override is specific: .dashboard h2, .dashboard h3, etc
